backend/gradle.properties

```diff
@@ -2,11 +2,9 @@
 
 exposed_version=0.28.1
 
-app.postgres.host=localhost
-app.postgres.port=5432
+app.postgres.url=jdbc:postgresql://localhost:5432/ehrenamtskarte
```

Comment:
didn't we have some problems with that?

Reply:
This is equivalent to the previous way. I no longer concat the url in java but pass the whole one here.